backend/copr_backend/storage.py

```diff
@@ -172,6 +180,11 @@ def delete_builds(self, dirname, chroot_builddirs, build_ids):
                         self.log.debug("can't remove %s", log_path)
         return result
 
+    def repository_exists(self, chroot):
+        repodata = os.path.join(self.opts.destdir, self.owner, self.project,
```

Comment:
Is `self.project` actually a `self.coprdir`?

Reply:
It's actually this:

https://github.com/fedora-copr/copr/blob/e123e4e16c0d1d3dfb17ff4610fc35d781292365/backend/copr_backend/actions.py#L100

which comes from frontend

https://github.com/fedora-copr/copr/blob/e123e4e16c0d1d3dfb17ff4610fc35d781292365/backend/copr_backend/actions.py#L100

so it should only be a project name, not CoprDir name. Which should IMHO be what we want because CoprDirs don't have their own repositories available (see #3369) and they use the main repo.